libraries/engage/favorites/components/Item/index.jsx

```diff
@@ -86,6 +87,7 @@ class Item extends Component {
                   <CTAButtons
                     productId={this.props.product.id}
                     active={this.state.visible}
+                    remove={this.props.remove}
```

Comment:
Only headline and image is linked to the product page. In PWA 7 the whole card is linked.

Reply:
fixed